.github/workflows/v8build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
               run: cd deps/depot_tools && git config --unset-all remote.origin.fetch; git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
               shell: bash
             - name: Build V8 linux
-              if: matrix.platform == 'ubuntu-latest'
+              if: matrix.platform == 'ubuntu-18.04'
```

Comment:
This can be replaced with `runner.os == 'Linux'`

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#runner-context

Reply:
Oooh, nice. Thanks @kuoruan 👍 

_.... now going to update all my projects_

Reply:
Do we want to change this before merge? if not, I can merge this PR now.

Reply:
I'll address this in another PR later, as we have already run the builds for this version